apps/spruce/src/pages/image/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import { Routes, Route, Navigate } from "react-router-dom";
+import { ImageTabRoutes } from "constants/routes";
+import useScrollToAnchor from "hooks/useScrollToAnchor";
+import { Header } from "./Header";
+import { BuildInformationTab } from "./tabs/index";
+
+type ImageTabsProps = {
+  imageId: string;
+  currentTab: ImageTabRoutes;
+};
+
+export const ImageTabs: React.FC<ImageTabsProps> = ({
+  currentTab,
+  imageId,
+}) => {
+  useScrollToAnchor();
+
+  return (
+    <Container>
+      <Header imageId={imageId} tab={currentTab} />
+      <Routes>
+        <Route
+          path="*"
+          element={<Navigate to={ImageTabRoutes.BuildInformation} replace />}
+        />
+        <Route
+          path={ImageTabRoutes.BuildInformation}
+          element={<BuildInformationTab imageId={imageId} />}
+        />
+      </Routes>
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  min-width: 600px;
+  width: 60%;
```

Comment:
What do you think of increasing the min width here? It's annoying that making the window smaller initially reduces the width of content instead of the width of whitespace

(would also be open to making this change for other settings pages but no need to address rn)

Reply:
not sure if I did quite what you asked but I made the table bigger!